core/src/verified_vote_packets.rs

```diff
@@ -3,20 +3,13 @@ use crate::{
     result::Result,
 };
 use solana_perf::packet::Packets;
-use std::{collections::HashMap, ops::Deref, time::Duration};
+use std::{collections::HashMap, time::Duration};
 
 #[derive(Default)]
 pub struct VerifiedVotePackets(HashMap<CrdsValueLabel, (u64, Packets)>);
 
-impl Deref for VerifiedVotePackets {
```

Comment:
Well, it turns out that we don't need wholesale `Deref` for `VerifiedVotePackets`, only modest `#[cfg(test)]` simpler getter called `get_vote_packets`.